### ISSUE TITLE
Minor UI Misalignment  -  Deployment to Dev Portal Pop-up Box

### ISSUE THREAD

**Issue by robincher** [getinsomnia/insomnia#4856]
### Expected Behavior

A message will shows that the yaml document is successfully deployed and available at the Dev Portal.

### Actual Behavior

The message intent is correct, just that there is a slight UI spacing misalignment as shown below in the screenshot.

<img width="457" alt="Screenshot 2022-06-13 at 3 52 52 PM" src="https://user-images.githubusercontent.com/328109/173310408-95c09cdc-5dcd-4210-92cf-0da4f60126e6.png">

### Reproduction Steps

1. From the Insomnia Dashboard, click the three dot menu on a Document. 
2. Select Deploy to Dev Portal and a modal will open. 
3. Enter the following information.
    - Kong API URL: http://localhost:8001 (admin api path)
    - Kong Workspace Name: Workspace name
    - Kong RBAC Token (optional): Leave it Blank if it is not set

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

I took a look at the code, and i think the its just a matter of edit in the following file.  Didn't have time to set-up my local env properly for development,so i thought i should highlight here first

plugins/insomnia-plugin-kong-portal/src/deploy-to-portal.tsx

```
   <div className="pad">
            <p className="no-pad no-margin-top">
              The Document is now available on
              {kongPortalLegacyMode ? (
                'Dev Portal'
              ) : (
                <a href={kongPortalUrl}>Dev Portal</a>
              )}
            </p>
            <div>
              <Button data-close-modal="true">Close</Button>
            </div>
  </div>
```

### Insomnia Version

2022.4.0

### What operating system are you using?

macOS

### Operating System Version

macOS Monterey 12.4

### Installation method

Downloaded from insomnia.rest

### Last Known Working Insomnia version

NA

**Comment by filfreire** [comment]
Hi @robincher, thank you for reporting this and for the additional information!

We have a fix for this ready on https://github.com/Kong/insomnia/pull/4858. Once it get's merged it should be available in one of our upcoming Insomnia Beta builds.

**Comment by robincher** [comment]
All right thanks @filfreire , will close this then :)

**Comment by jacksjm** [comment]
@filfreire this issue was not fixed in #5069?
